frontend/catalyst/oqc/src/CMakeLists.txt

```
@@ -0,0 +1,56 @@
+cmake_minimum_required(VERSION 3.20)
+
+project(catalyst_oqc)
+
+set(runtime_includes "${PROJECT_SOURCE_DIR}/../../../../runtime/include")
+set(backend_includes "${PROJECT_SOURCE_DIR}/../../../../runtime/lib/backend/common")
+set(util_includes "${PROJECT_SOURCE_DIR}/../../../../runtime/utils")
+set(runtime_lib "${RUNTIME_BUILD_DIR}/lib/libcatalyst_python_interpreter.so")
```

I'll think a bit more about how to set the rpath correctly so that it looks for the library not in the build directory but in the installed directory. That is what I imagine third parties will like. But the below is needed in conjunction with another change below to avoid the error:

```
python: symbol lookup error: frontend/catalyst/oqc/src/build//librtd_oqc.so: undefined symbol: _Z14getPythonMutexv
```

```suggestion
set(runtime_lib "${RUNTIME_BUILD_DIR}/lib")
```